app/code/Magento/InventoryInStorePickupApi/Model/Mapper/CreateFromSourceInterface.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryInStorePickupApi\Model\Mapper;
+
+use Magento\InventoryApi\Api\Data\SourceInterface;
+use Magento\InventoryInStorePickupApi\Api\Data\PickupLocationInterface;
+
+/**
+ * Create Pickup Location based on Source (Service Provider Interface - SPI).
```

Comment:
Generally we do not differentiate API / SPI even if you may have some ideas on it's usage scenarios.

Reply:
Just removed this part of comment to make description more clear.